root/relationship/linktype/form.tt

```diff
@@ -1,55 +1,55 @@
+        [% r.label('child_order', add_colon('Child order')) %]
         [% r.number('child_order') %]
         [% field_errors(form, 'child_order') %]
     [% END %]
 
-    [% form_row_text_long(r, 'name', add_colon(l('Name'))) %]
-    [% form_row_text_long(r, 'link_phrase', l('Forward link phrase:')) %]
-    [% form_row_text_long(r, 'reverse_link_phrase', l('Reverse link phrase:')) %]
-    [% form_row_text_long(r, 'long_link_phrase', l('Long link phrase:')) %]
-    [% form_row_checkbox(r, 'is_deprecated', l('This relationship is deprecated')) %]
-    [% form_row_checkbox(r, 'has_dates', l('This relationship allows setting dates')) %]
+    [% form_row_text_long(r, 'name', add_colon('Name')) %]
+    [% form_row_text_long(r, 'link_phrase', 'Forward link phrase:') %]
+    [% form_row_text_long(r, 'reverse_link_phrase', 'Reverse link phrase:') %]
+    [% form_row_text_long(r, 'long_link_phrase', 'Long link phrase:') %]
+    [% form_row_checkbox(r, 'is_deprecated', 'This relationship is deprecated') %]
+    [% form_row_checkbox(r, 'has_dates', 'This relationship allows setting dates') %]
 
     [% WRAPPER form_row %]
-        [% r.label('description', add_colon(l('Description'))) %]
+        [% r.label('description', add_colon('Description')) %]
         [% r.textarea('description', { cols => 80, rows => 6 }) %]
         [% field_errors(form, 'description') %]
     [% END %]
 
     [% WRAPPER form_row %]
-        [% r.label('entity0_cardinality', add_colon(l('Cardinality of {entity_placeholder}', { entity_placeholder => '<code>{entity0}</code>' }))) %]
+        [% r.label('entity0_cardinality', add_colon('Cardinality of {entity_placeholder}', { entity_placeholder => '<code>{entity0}</code>' })) %]
         [% r.number('entity0_cardinality') %]
         [% field_errors(form, 'entity0_cardinality') %]
     [% END %]
     [% WRAPPER form_row %]
-        [% r.label('entity1_cardinality', add_colon(l('Cardinality of {entity_placeholder}', { entity_placeholder => '<code>{entity1}</code>' }))) %]
+        [% r.label('entity1_cardinality', add_colon('Cardinality of {entity_placeholder}', { entity_placeholder => '<code>{entity1}</code>' })) %]
         [% r.number('entity1_cardinality') %]
         [% field_errors(form, 'entity1_cardinality') %]
     [% END %]
 
     [% WRAPPER form_row %]
-        [% r.label('orderable_direction', l('Orderable direction:')) %]
+        [% r.label('orderable_direction', 'Orderable direction:') %]
         [% r.number('orderable_direction') %]
         <div class="no-label">
-            [% l('A value of 1 or 2 indicates that either entity1 or entity0 are orderable, respectively.') %]
+            [% 'A value of 1 or 2 indicates that either entity1 or entity0 are orderable, respectively.' %]
         </div>
         [% field_errors(form, 'orderable_direction') %]
     [% END %]
 
     [% WRAPPER form_row %]
-        [% r.label('priority', l('Priority:')) %]
+        [% r.label('priority', 'Priority:') %]
```

Comment:
Same

Reply:
This is an undocumented field and seems like it's entirely unused, so we'll probably just drop it with MBS-13403 - as such, not worth keeping I don't think.